crates/bevy_pbr/src/fog.rs

```diff
@@ -0,0 +1,488 @@
+    ///
+    /// ## Formula
+    ///
+    /// The fog intensity for a given point in the scene is determined by the following formula:
+    ///
+    /// ```text
+    /// let fog_intensity = 1.0 - 1.0 / (distance * density).exp();
+    /// ```
+    ///
+    /// <svg width="370" height="212" viewBox="0 0 370 212" fill="none">
+    /// <title>Plot showing how exponential fog falloff behaves for different density values</title>
+    /// <mask id="mask0_3_31" style="mask-type:alpha" maskUnits="userSpaceOnUse" x="42" y="42" width="286" height="108">
+    /// <rect x="42" y="42" width="286" height="108" fill="#D9D9D9"/>
+    /// </mask>
+    /// <g mask="url(#mask0_3_31)">
+    /// <path d="M42 150C42 150 98.3894 53 254.825 53L662 53" stroke="#FF003D" stroke-width="1"/>
+    /// <path d="M42 150C42 150 139.499 53 409.981 53L1114 53" stroke="#001AFF" stroke-width="1"/>
+    /// <path d="M42 150C42 150 206.348 53 662.281 53L1849 53" stroke="#14FF00" stroke-width="1"/>
+    /// </g>
+    /// <path d="M331 151H42V49" stroke="currentColor" stroke-width="2"/>
+    /// <text font-family="sans-serif" fill="currentColor" style="white-space: pre" font-size="12" letter-spacing="0em"><tspan x="136" y="173.864">1</tspan></text>
+    /// <text font-family="sans-serif" fill="currentColor" style="white-space: pre" font-size="12" letter-spacing="0em"><tspan x="30" y="53.8636">1</tspan></text>
+    /// <text font-family="sans-serif" fill="currentColor" style="white-space: pre" font-size="12" letter-spacing="0em"><tspan x="42" y="173.864">0</tspan></text>
+    /// <text font-family="sans-serif" fill="currentColor" style="white-space: pre" font-size="12" letter-spacing="0em"><tspan x="232" y="173.864">2</tspan></text>
+    /// <text font-family="sans-serif" fill="currentColor" style="white-space: pre" font-size="12" letter-spacing="0em"><tspan x="332" y="173.864">3</tspan></text>
+    /// <text font-family="sans-serif" fill="#FF003D" style="white-space: pre" font-size="10" letter-spacing="0em"><tspan x="77" y="64.6364">density = 2</tspan></text>
+    /// <text font-family="sans-serif" fill="#001AFF" style="white-space: pre" font-size="10" letter-spacing="0em"><tspan x="236" y="76.6364">density = 1</tspan></text>
+    /// <text font-family="sans-serif" fill="#14FF00" style="white-space: pre" font-size="10" letter-spacing="0em"><tspan x="205" y="115.636">density = 0.5</tspan></text>
+    /// <text font-family="sans-serif" fill="currentColor" style="white-space: pre" font-size="12" letter-spacing="0em"><tspan x="161" y="190.864">distance</tspan></text>
+    /// <text font-family="sans-serif" transform="translate(10 132) rotate(-90)" fill="currentColor" style="white-space: pre" font-size="12" letter-spacing="0em"><tspan x="0" y="11.8636">fog intensity</tspan></text>
+    /// </svg>
+    Exponential {
+        /// Unitless multiplier applied to the world distance (within the exponential fog falloff calculation).
```

Comment:
Is it really unitless? Density suggests some measure of mass or matter per volume. Do you just mean it is not physically-based?

Reply:
You're right, it's _not_ unitless, I think I wrote this before I had a good understanding of what the math was actually doing, and missed it when revising the docs. If I understand it correctly, the unit should be $m^{-1}$. It's also _kinda_ physically based, less so than the atmospheric fog, but still an approximation/simplification. I'll just remove the word `Unitless`.